docs/system_overview/configuration.md

```diff
@@ -14,244 +14,640 @@ This scrambled password file is saved after an `iinit` is run. If this file does
+    // This must be the same across all iRODS servers in a Zone.
+    "irods_server_control_plane_encryption_algorithm": "AES-256-CBC",
+
+    // (Optional)
+    // The number of hash rounds used in the control plane communications.
+    // This must be the same across all iRODS servers in a Zone.
+    "irods_server_control_plane_encryption_num_hash_rounds": 16,
+
+    // (Optional)
+    // The encryption key required for communicating with the iRODS grid control plane.
+    "irods_server_control_plane_key": "32_byte_server_control_plane_key",
+
+    // (Optional)
+    // The port number on which the control plane listens.
+    "irods_server_control_plane_port": 1248,
+
+    // (Optional)
+    // Location of a file of trusted CA certificates in PEM format.
+    // Note that the certificates in this file are used in conjunction with the system
+    // default trusted certificates.
+    "irods_ssl_ca_certificate_file": "",
+
+    // (Optional)
+    // Location of a directory containing CA certificates in PEM format.
+    // The files each contain one CA certificate. The files are looked up by the CA subject name hash
+    // value, which must hence be available. If more than one CA certificate with the same name hash
+    // value exist, the extension must be different (e.g. 9d66eef0.0, 9d66eef0.1 etc). The search is
+    // performed in the ordering of the extension number, regardless of other properties of the
+    // certificates. Use the ‘c_rehash’ utility to create the necessary links.
+    "irods_ssl_ca_certificate_path": "",
+
+    // (Optional)
+    // The file containing the server's certificate chain.
+    // The certificates must be in PEM format and must be sorted starting with the subject's
+    // certificate (actual client or server certificate), followed by intermediate CA certificates
+    // if applicable, and ending at the highest level (root) CA.
+    "irods_ssl_certificate_chain_file": "",
+
+    // (Optional)
+    // The private key corresponding to the server's certificate in the certificate chain file.
+    "irods_ssl_certificate_key_file": "",
+
+    // (Optional)
+    // The Diffie-Hellman parameter file location.
+    "irods_ssl_dh_params_file": "",
+
+    // (Optional)
+    // TODO
+    "irods_transfer_buffer_size_for_parallel_transfer_in_megabytes": 4,
+
+    // The username within iRODS for this account.
+    "irods_user_name": "",
+
+    // The name of the iRODS Zone.
+    "irods_zone_name": "",
+
+    // (Optional)
+    // Defines the filename of the JSON schema used for validation when not specified explicitly.
+    // Only applies to service account environment files.
+    "schema_name": "service_account_environment",
+
+    // (Optional)
+    // Defines the version of the schema used for validation.
+    // Only applies to service account environment files.
+    "schema_version": "v4"
+}
+```
+
+A client environment file might look like this.
+
+```json
+{
+    "irods_host": "node-1.example.company.com",
```

Comment:
let's stick to example.org as the root domain, since it's designed for this usage (documentation)

Reply:
done